tests/queries/0_stateless/01509_format_raw_blob.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../shell_config.sh
+
+${CLICKHOUSE_CLIENT} -n --query "
+DROP TABLE IF EXISTS t;
+CREATE TABLE t (a LowCardinality(Nullable(String))) ENGINE = Memory;
+"
+
+${CLICKHOUSE_CLIENT} --query "INSERT INTO t FORMAT RawBLOB" < ${BASH_SOURCE[0]}
+
+cat ${BASH_SOURCE[0]} | md5sum
```

Comment:
Nice :)

Reply:
Maybe we can use it instead of  `nc` and `scp`, `rsync`..